Services/DataBaseService/Repositories/UserCredentialRepository.cs

```diff
@@ -31,6 +31,16 @@ public void Create(UserEmailPassword data)
                 throw new Exception(e.Message + "\nCan`t add user");       
             }
         }
+
+        public void Delete(Guid data)
+        {
+            var user = dbContext.Find<DbUserCredential>(data);
+            if (user!=null)
```

Comment:
`(!(user is null))` is recommended
https://www.gullberg.tk/blog/is-null-versus-null-in-c/